docs/troubleshooting/faq.rst

```diff
@@ -0,0 +1,16 @@
+Frequently Asked Questions
+==========================
+
+I stopped and started my VM, and now JupyterHub doesn't work!

```

Comment:
I think we should also be a lot more specific here, since there is a lot of reasons this could be the case!

Maybe phrase it something else than "Doesn't work" and focus it on Google Cloud?

Reply:
yeah, perhaps this section should be in the google cloud part of the guide in general, instead of in faq?

Reply:
yeah. But not the tutorial though, I think the tutorial should all be the same structure. Maybe start a provider specific tips section of the guides?